apps/api/views/homework.py

```diff
@@ -32,11 +36,25 @@ class SubmissionViewSet(ModelViewSet):
     serializer_class = SubmissionSerializer
     permission_classes = (SubmissionPermissionCheck,)
 
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        self.perform_create(serializer)
+        headers = self.get_success_headers(serializer.data)
+        return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
+
     def perform_create(self, serializer):
         new_sub = serializer.save()
         if new_sub.pk and not new_sub.submitted_to_challenge:
-            if not new_sub.definition.questions_only:
-                post_submission(new_sub.pk)
+            if not new_sub.definition.questions_only and new_sub.github_url:
+                logger.info("Submission is a github submission!")
```

Comment:
i think we could add the PK here or something:

```python
logger.info(f"Submission={new_sub.pk} is a github submission!")
```

Reply:
👍